workflow/scripts/get_pathway_barplot.py

```diff
@@ -0,0 +1,15 @@
+import altair as alt
```

Comment:
Script can be removed in favor of the datavzd report.

Reply:
Noted